packages/tre/src/index.ts

```diff
@@ -84,6 +85,31 @@ function validateOptions(
   return [pluginSharedOpts, pluginWorkerOpts];
 }
 
+// When creating user worker services, we need to know which Durable Objects
+// they export. Rather than parsing JavaScript to search for class exports
+// (which would have to be recursive because of `export * from ...`), we collect
+// all Durable Object bindings, noting that bindings may be defined for objects
+// in other services.
+function getDurableObjectClassNames(
+  allWorkerOpts: PluginWorkerOptions[]
+): Map<string, string[]> {
+  const serviceClassNames = new Map<string, string[]>();
+  for (const workerOpts of allWorkerOpts) {
+    const workerServiceName = getUserServiceName(workerOpts.core.name);
+    for (const klass of Object.values(workerOpts.do.durableObjects ?? {})) {
```

Comment:
Maybe something more descriptive than `klass`? I know you can't use `class`, but maybe `durableObject`?